test/pxci.test.js

```diff
@@ -0,0 +1,49 @@
+const should = require('should');
+const { V2CredentialsIntelligenceProtocol } = require('../lib/extract_field/credentials_intelligence/V2CredentialsIntelligenceProtocol');
+
+describe('Check Credentials Intelligent v2 hashing', function() {
+    it('Should hash username', (done) => {
+        const credentialsBody = { username: 'pxUser', password: '1234' };
```

Comment:
Why generate a "credentialsBody" object? You can keep username and password separate here since you pass them to the `ProcessCredentials` function separately anyway.